src/en/ohjoysextoy/src/eu/kanade/tachiyomi/extension/en/ohjoysextoy/OhJoySexToy.kt.kt

```diff
@@ -0,0 +1,132 @@
+import keiyoushi.utils.tryParse
+import okhttp3.Request
+import okhttp3.Response
+import org.jsoup.nodes.Document
+import org.jsoup.nodes.Element
+import java.text.SimpleDateFormat
+import java.util.Locale
+
+class OhJoySexToy : ParsedHttpSource() {
+
+    override val name = "Oh Joy Sex Toy"
+    override val baseUrl = "https://www.ohjoysextoy.com"
+    override val lang = "en"
+    override val supportsLatest = true
+
+    // Browse
+
+    override fun popularMangaRequest(page: Int): Request = GET("$baseUrl/category/comic/page/$page/")
+
+    override fun popularMangaSelector(): String = ".comicthumbwrap"
+
+    override fun popularMangaFromElement(element: Element): SManga = SManga.create().apply {
+        setUrlWithoutDomain(element.select(".comicarchiveframe > a").attr("href"))
+        title = element.selectFirst(".comicthumbdate")!!.text().substringBefore(" by")
+        thumbnail_url = element.selectFirst(".comicarchiveframe > a > img")!!.attr("src")
+    }
+
+    override fun popularMangaNextPageSelector(): String = ".pagenav-left a"
+
+    // Latest
+
+    override fun latestUpdatesRequest(page: Int): Request = GET(baseUrl)
+
+    override fun latestUpdatesSelector(): String = "#MattsRecentComicsBar > ul > div"
+
+    override fun latestUpdatesFromElement(element: Element): SManga = SManga.create().apply {
+        setUrlWithoutDomain(element.select(".comicarchiveframe > a").attr("href"))
+        title = element.selectFirst(".comicthumbdate")!!.text().substringBefore(" by")
+        thumbnail_url = element.selectFirst(".comicarchiveframe > a > img")!!.attr("src")
+    }
+
+    override fun latestUpdatesNextPageSelector(): String? = null
+
+    // Search
+
+    override fun searchMangaRequest(page: Int, query: String, filters: FilterList): Request = GET("$baseUrl/?s=$query")
+
+    override fun searchMangaSelector(): String = "h2.post-title"
+
+    override fun searchMangaFromElement(element: Element): SManga = SManga.create().apply {
+        setUrlWithoutDomain(element.selectFirst("a")!!.attr("href"))
+        title = element.selectFirst("a")!!.text().substringBefore(" by")
+    }
+
+    override fun searchMangaNextPageSelector(): String? = null
+
+    // etc
+
+    override fun mangaDetailsParse(document: Document): SManga = SManga.create().apply {
+        thumbnail_url = document.selectFirst("meta[property=\"og:image\"]")
+            ?.attr("content")
+            ?: ""
+        status = SManga.COMPLETED
+        title = document.selectFirst("meta[property=\"og:title\"]")
+            ?.attr("content")
+            ?.substringBefore(" by")
+            ?: ""
+        author = document.selectFirst("meta[property=\"og:title\"]")
+            ?.attr("content")
+            ?.substringAfter("by ")
+            ?: ""
+        description = parseDescription(document)
+        genre = document.select("meta[property=\"article:section\"]:not(:first-of-type)")
+            .eachAttr("content")
+            .joinToString(", ")
```

Comment:
I think this is the default
```suggestion
            .joinToString()
```